### PR TITLE
Miru button component docs

### PR DESCRIPTION
**Pull request by prasanthchaduvula**

### CODE REVIEW

File: docs/docs/contributing-guide/components/button.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#### Using the Button Component
+
+The Button component can be used with various props to control its appearance.
+Here are some examples of how we can use it:
+
+```
+<Button
+  style="secondary"
+>
+  Button
```
Comment (apoorv1316) [inline docs/docs/contributing-guide/components/button.md:31]: @prasanthchaduvula The text 'Button' seems a little bit confusing as its already wrapped b/w Button. Can we change the text to something else?

File: docs/docs/contributing-guide/components/button.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+</Button>
+```
+
+![default btn](/img/button/secondary-btn.png)
+
+```
+<Button
+  style="ternary"
+>
+  Button
```
Comment (apoorv1316) [inline docs/docs/contributing-guide/components/button.md:41]: here too

File: docs/docs/contributing-guide/components/button.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+![default btn](/img/button/ternary-btn.png)
+
+#### Button Component Props
+
+| Name        | Description                                                              | Default Value     |
+| ----------- | ------------------------------------------------------------------------ | ----------------- |
+| `style`     | To specify the style of the Button. `"primary", "secondary", "ternary" ` | `primary`         |
+| `onClick`   | To specify the action to be triggered on clicking the Button. `func`     | `"(e) => (void)"` |
+| `size`      | To set the size of the Button. `"small", "medium", "large"`              | `"medium"`        |
+| `disabled`  | To set Button as disabled `bool`                                         | `false`           |
```
Comment (apoorv1316) [inline docs/docs/contributing-guide/components/button.md:54]: Shouldn't it be 'boolean'?

**Review by apoorv1316 — CHANGES_REQUESTED** [review]

File: docs/docs/contributing-guide/components/button.md (hunk 3 above)
Comment (prasanthchaduvula) [reply docs/docs/contributing-guide/components/button.md:54]: Yes, written `bool` as shorthand to boolean and `func` as shorthand to function

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**